src/index.d.ts

```diff
@@ -227,6 +227,11 @@ declare namespace preact {
 		vnode: ComponentChild,
 		parent: Element | Document | ShadowRoot | DocumentFragment
 	): void;
+	function cloneElement(
+		vnode: preact.ComponentChild,
```

Comment:
Is it possible to set `element` argument of `cloneElement()` in React to `string` or `number` or `boolean` or `null` or `undefined`?

Reply:
Haven't tried it in React but in React + TS, I think one is able to pass in anything to `cloneElement(element)` given that `element` is `props.children`. 

Although directly cloning one of those primitives is not possible (e.g. `cloneElement("my string")`)

Reply:
`createElement()` returns only `VNode`. 
But, this `cloneElement()` handles `VNode` and others. 
This is inconsistance of function name.